### ISSUE TITLE
Not compatible with e10s

### ISSUE THREAD

**Issue by J-Rey** [Dagger0/IPvFox#14]
Its not compatible for 2 reasons:
1. The icon on the toolbar doesn't show any hosts anymore & just says "No hosts seen yet" when clicking on it so core functionality is lost.
2. Mozilla is requiring extension signing now as described with their [Extension Signing page](https://wiki.mozilla.org/Addons/Extension_Signing).

**Comment by Dagger0** [comment]
The extension signing is entirely out of my hands. Mozilla will sign the version on AMO automatically, and if you want to run it from the repo you'll have to run one of the special non-release versions of Firefox that actually let you do that. Yes, this sucks -- take it up with Mozilla.

Not working on Nightly is because you have e10s enabled. I'm waiting on https://bugzilla.mozilla.org/show_bug.cgi?id=1108827 before I try to do anything about this. (Although I've yet to get e10s to work for me at all, so that's going to be fun.)

**Comment by J-Rey** [comment]
You're right about multi-process (e10s) being the cause of the lost functionality but its coming up so hopefully you can get it fixed in your code or they can get a workaround (like the bug is about).

The warning in Add-ons Manager about this add-on possibly not being compatible & referring to the extension signing link is gone now (with me just keeping the Dev Edition updated) & it even says is signed after the version number.

Thanks for all this!

**Comment by nivex** [comment]
The referenced bug in the second comment got closed as WONTFIX. Chatting with Dagger on IRC he suggested that https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/WebRequest.jsm may be of some help here. I know he's not all that interested in fixing this, so I'm leaving this here in case someone else gets inspired to look into it.
